src/KeyFrame.cpp

```diff
@@ -55,7 +55,7 @@ void Keyframe::ReorderPoints() {
 	}
 }
 
-// Constructor which sets the default point & coordinate at X=0
+// Constructor which sets the default point & coordinate at X=1
```

Comment:
This one... I'm _really_ ambivalent about. It's _not_ as immediately obvious from reading the code, so I suppose it's arguably useful. (It's certainly better that it at least be _**correct**_, though, so thanks for taking the time to clean these up!)